lib/matplotlib/texmanager.py

```diff
@@ -92,19 +104,24 @@ def get_font_config(self):
         fontconfig = [self._font_family]
         fonts = {}
         for font_family in self._font_families:
-            for font in rcParams['font.' + font_family]:
-                if font.lower() in self._font_info:
-                    fonts[font_family] = self._font_info[font.lower()]
-                    _log.debug(
-                        'family: %s, font: %s, info: %s',
-                        font_family, font, self._font_info[font.lower()])
-                    break
-                else:
-                    _log.debug('%s font is not compatible with usetex.', font)
+            if reduced_notation and self._font_family == font_family:
+                fonts[font_family] = self._font_info[ff_val]
             else:
-                _log.info('No LaTeX-compatible font found for the %s font '
-                          'family in rcParams. Using default.', font_family)
-                fonts[font_family] = self._font_info[font_family]
+                for font in rcParams['font.' + font_family]:
+                    if font.lower() in self._font_info:
+                        fonts[font_family] = self._font_info[font.lower()]
+                        _log.debug(
+                            'family: %s, font: %s, info: %s',
+                            font_family, font, self._font_info[font.lower()])
+                        break
+                    else:
+                        _log.debug('%s font is not compatible with usetex.',
+                                    font)
+                else:
+                    _log.info('No LaTeX-compatible font found for the %s font'
+                                'family in rcParams. Using default.',
+                                font_family)
```

Comment:
Align with opening parenthesis.
```suggestion
                              'family in rcParams. Using default.',
                              font_family)
```